pyproject.toml

```diff
@@ -35,7 +35,7 @@ Releases = "https://nvidia.github.io/cuda-quantum/latest/releases.html"
 # We must use h5py<3.11 because 3.11 doesn't include aarch64 Linux wheels.
 # https://github.com/h5py/h5py/issues/2408
 [project.optional-dependencies]
-chemistry = [ "scipy==1.10.1", "openfermionpyscf==0.5", "h5py<3.11" ]
+chemistry = [ "scipy==1.10.1", "openfermionpyscf==0.5", "h5py<3.11", "qutip>=5" ]
```

Comment:
The new dependency is not strictly related to "chemistry", so better to separate it. Alternatively, you can remove it from dependency and rely on users to explicitly install it.

Reply:
I've separated this to a new dependency in the new commit.